src/NuGet.Core/NuGet.CommandLine.XPlat/ReportRenderers/Interfaces/IReportRenderer.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal interface IReportRenderer
+    {
+        void AddProblem(string errorText, ProblemType problemType);
+        IEnumerable<ReportProblem> GetProblems();
+        void End(ListPackageReportModel reportProject);
```

Comment:
AddProjectReport instead of End? 
The `End` name seems like something that was originally end in the implementation, but it has evolved now.

Reply:
The interface is already callled Renderer, so AddToRenderer doesn't really say much. 

I think it should describe what you're adding instead.

Reply:
Ok. Renamed to `AddProjectReport`, please note this method also trigger the actual rendering to console/json.